From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sun, 17 Apr 2016 15:37:38 +0200
Subject: [PATCH] bootstrap: Add armband-rtc-efi-fix package.

Older ThunderX and possibly other UEFI-enabled targets do not support
rtc-efi properly, so they end up filling dmesg with useless
complaints about not being able to read system time.

armband-rtc-efi-fix validates rtc-efi can be used on the system,
otherwise it rmmods rtc_efi kernel module via rcS script.

This patch hardcodes the addition of armband-rtc-ef-fix package during
bootstrap image build.

NOTES:
This could be later removed or, if other fixes need to be delivered
like this, grouped in a meta package.

Fuel 9.0 moved default package list to openstack.yaml fixture, see [1].

[1] https://github.com/openstack/fuel-web/commit/
    4ee42effe27694bd231663e3d0f10c0c42877177
---
 build/f_isoroot/f_repobuild/fuel_bootstrap_cli.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build/f_isoroot/f_repobuild/fuel_bootstrap_cli.yaml b/build/f_isoroot/f_repobuild/fuel_bootstrap_cli.yaml
index 2e4843d..15d9e9d 100644
--- a/build/f_isoroot/f_repobuild/fuel_bootstrap_cli.yaml
+++ b/build/f_isoroot/f_repobuild/fuel_bootstrap_cli.yaml
@@ -28,6 +28,7 @@
     - xz-utils
     - squashfs-tools
     - msmtp-mta
+    - armband-rtc-efi-fix
   bootstrap_images_dir: /var/www/nailgun/bootstraps
   active_bootstrap_symlink: /var/www/nailgun/bootstraps/active_bootstrap
   flavor: ubuntu
